scripts/fedora/openstack.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash -eux
 
-if [ $(uname -m)=="ppc64" -o $(uname -m)=="ppc64le" ]
-  then
-    dnf -y install ppc64-diag
-fi
-
 if [ -e /usr/bin/dnf ] ; then
   dnf -y install cloud-init cloud-utils yum
```

Comment:
Is there a reason we install `yum` when `dnf` is present already ?
I thought `dnf` is better than `yum` in many aspects. Chef support was lacking last I saw an year ago but should have been fixed by now.